gfx/NeoDays/pngs_tiles_10x10/terrain/t_shrub_blueberry_harvested.json

```diff
@@ -0,0 +1,8 @@
+{
+  "id": [
+    "t_shrub_blueberry_harvested",
+    "t_shrub_blueberry_season_spring",
```

Comment:
same

Reply:
This one is required, otherwise the shrubs fallback to the unharvested sprite.